runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -281,6 +313,9 @@ public UUID getStreamID() {
                 catch (Exception ex) {}
                 sleepTime = min(sleepTime * 2L, maxSleepTime);
                 retries++;
+            } catch (Exception e) {
+                log.debug("TXExecute[{}] Exception: {}", this, e);
```

Comment:
Add that we are aborting this exception (TXExecute[{}] Abort with Exception: {})